src/templates/admin/vocgui/delete_confirmation.html

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
@lukasboehm97 is it necessary that the templates reside in the `src` directory directly instead of a subdirectory of an app?

Reply:
Yes, it is. It is recommended by Django for overriding admin templates (it gets detected by default using BASE_DIR)